docs/intro/tutorial06.txt

```diff
@@ -2,119 +2,438 @@
-    }
-
-Reload ``http://localhost:8000/polls/`` and you should see the background
-loaded in the bottom right of the screen.
-
-.. warning::
-
-    Of course the ``{% static %}`` template tag is not available for use in
-    static files like your stylesheet which aren't generated by Django. You
-    should always use **relative paths** to link your static files between each
-    other, because then you can change :setting:`STATIC_URL` (used by the
-    :ttag:`static` template tag to generate its URLs) without having to modify
-    a bunch of paths in your static files as well.
+Take a few minutes to marvel at all the code you didn't have to write. By
+registering the ``Question`` model with ``admin.site.register(Question)``,
+Django was able to construct a default form representation. Often, you'll want
+to customize how the admin form looks and works. You'll do this by telling
+Django the options you want when you register the object.
 
-These are the **basics**. For more details on settings and other bits included
-with the framework see
-:doc:`the static files howto </howto/static-files/index>` and
-:doc:`the staticfiles reference </ref/contrib/staticfiles>`. :doc:`Deploying
-static files </howto/static-files/deployment>` discusses how to use static
-files on a real server.
+Let's see how this works by re-ordering the fields on the edit form. Replace
+the ``admin.site.register(Question)`` line with:
+
+.. snippet::
+    :filename: polls/admin.py
+
+    from django.contrib import admin
+
+    from .models import Question
+
+
+    class QuestionAdmin(admin.ModelAdmin):
+        fields = ['pub_date', 'question_text']
+
+    admin.site.register(Question, QuestionAdmin)
+
+You'll follow this pattern -- create a model admin class, then pass it as the
+second argument to ``admin.site.register()`` -- any time you need to change the
+admin options for an model.
+
+This particular change above makes the "Publication date" come before the
+"Question" field:
+
+.. image:: _images/admin07.png
+   :alt: Fields have been reordered
+
+This isn't impressive with only two fields, but for admin forms with dozens
+of fields, choosing an intuitive order is an important usability detail.
+
+And speaking of forms with dozens of fields, you might want to split the form
+up into fieldsets:
+
+.. snippet::
+    :filename: polls/admin.py
+
+    from django.contrib import admin
+
+    from .models import Question
+
+
+    class QuestionAdmin(admin.ModelAdmin):
+        fieldsets = [
+            (None,               {'fields': ['question_text']}),
+            ('Date information', {'fields': ['pub_date']}),
+        ]
+
+    admin.site.register(Question, QuestionAdmin)
+
+The first element of each tuple in
+:attr:`~django.contrib.admin.ModelAdmin.fieldsets` is the title of the fieldset.
+Here's what our form looks like now:
+
+.. image:: _images/admin08t.png
+   :alt: Form has fieldsets now
+
+You can assign arbitrary HTML classes to each fieldset. Django provides a
```

Comment:
I'd like to remove the stuff about "arbitrary HTML classes" and "collapse" -- it's really annoying to have required fields collapsed by default and this isn't vital info the tutorial needs to cover.